agreement/cryptoVerifier.go

```diff
@@ -214,15 +214,7 @@ func (c *poolCryptoVerifier) voteFillWorker(toBundleWait chan<- bundleFuture) {
 			}
 
 			uv := votereq.message.UnauthenticatedVote
-			err := c.voteVerifier.verifyVote(votereq.ctx, c.ledger, uv, votereq.TaskIndex, votereq.message, c.votes.out)
-			if err != nil && c.votes.out != nil {
-				select {
-				case c.votes.out <- asyncVerifyVoteResponse{index: votereq.TaskIndex, err: err, cancelled: true}:
-				default:
-					voteVerifierOutFullCounter.Inc(nil)
```

Comment:
This is the only place where feeding to` c.votes.out` is not blocked and `voteVerifierOutFullCounter` is incremented. 
`voteVerifierOutFullCounter` is not used elsewhere.